examples/rendy/main.rs

```diff
@@ -594,16 +594,27 @@ fn main() -> amethyst::Result<()> {
     )?;
 
     let game_data = GameDataBuilder::default()
+        .with_bundle(
+            RenderingBundle::<DefaultBackend>::new()
```

Comment:
This is not right. `RenderingBundle` must always be added at the end, because it adds a barrier at the beginning and expects all game data to be already written. This change adds another frame of delay to the rendering. What should happen instead is that systems should not rely on `Window` being initialized at the setup time. I've made sure of that while introducing RenderingBundle before, in fact it worked like that before specs update. Please make sure that some changes weren't overwritten in that process.